Moli/Core/Screens/EditProfile/SocialFieldView.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct SocialFieldView: View {
+    var label: String
+    var imageName: String
+
+    var body: some View {
+        RoundedRectangle(cornerRadius: 8, style: .continuous)
+            .fill(Color.gray100)
+            .frame(height: 48)
+            .overlay {
+                HStack {
+                    Text(label)
+                        .font(.labelSmall)
```

Comment:
:art: Det här är en superficiell men såg i Designen att det inte fanns någon exakt font utvald, men såg att attributerna var "Regular" och "14px", skillnaden mellan labelSmall och paragraphSmall är att labelSmall har by default "Normal" och "14px", checka ut `Fonts.swift` filen :) det här är en update som jag gjorde på fonterna efter ett snack med Aida om font systemet

```suggestion
                        .font(.paragraphSmall)
```